src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -163,6 +163,11 @@ void SolverInterfaceImpl::configure(
   if (_accessorProcessRank == 0) {
     PRECICE_INFO("This is preCICE version " << PRECICE_VERSION);
     PRECICE_INFO("Revision info: " << precice::preciceRevision);
+#ifndef NDEBUG
+    PRECICE_INFO("Configuration: Release (Debug and Trace log unavailable)");
```

Comment:
Problem with this is that it is again only a partial solution. It will only work if the user wants Debug or Trace output for `SolverInterfaceImpl` specifically (or everywhere).

Reply:
We can implement this without the disabled macros. For this, we need to directly invoke the logger.
So something along these lines:

```cpp
_log.debug(PRECICE_LOG_LOCATION, "Debug log is not available in a Release build. Build preCICE again in Debug mode to see debug messages.");
_log.debug(PRECICE_LOG_LOCATION, "Trace log is not available in a Release build. Build preCICE again in Debug mode to see trace messages.");
```